hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BootstrapConfig.java

```diff
@@ -48,5 +48,5 @@ public record BootstrapConfig(
         @ConfigProperty(value = "rates.nextExpiry", defaultValue = "4102444800") @NetworkProperty long ratesNextExpiry,
         @ConfigProperty(value = "system.entityExpiry", defaultValue = "1812637686") @NetworkProperty
                 long systemEntityExpiry,
-        @ConfigProperty(value = "throttleDefsJson.resource", defaultValue = "throttles.json") @NodeProperty
+        @ConfigProperty(value = "throttleDefsJson.resource", defaultValue = "throttles-dev.json") @NodeProperty
```

Comment:
Looks like the mono is using the `throttles-dev.json` instead of `throttles.json`. `UmbrellaRedux.umbrellaRedux` test was failing due to configuration differences - mainly because of the `CryptoCreate`. In `throttles-dev.json` the _milliOpsPerSec_ value is set to **2000000**, in `throttles.json` to **2000**.

@MiroslavGatsanoga suggested another option - change the test to load the correct config with 
`protoDefsFromResource("testSystemFiles/throttles-dev.json")`  - with that change the test is also passing.

@tinker-michaelj @netopyr thoughts?

Reply:
What are we using in production currently? The default value of any `ConfigProperty` should reflect that.

Reply:
I'm not sure exactly which of all configs are for production but I doubt that the value is `throttles-dev.json` 😄 I can revert this change but is it okay to change the suite to use the dev throttle file?

Reply:
I just checked the default in `BootstrapConfig` has to be `throttles.json`.

How do you see that the tests in mono are using `throttles-dev.json`? I think it makes sense to update the test and make the usage of `throttles-dev.json` more explicit.

Reply:
Thanks, Miro. I wasn't sure exactly which property file was using mono but when I debugged it I saw that the throttle values are as `throttles-dev.json`.

I will revert the BootstrapConfig change and I will update the test 👍